src/Serilog.Sinks.Elasticsearch/Serilog.Sinks.ElasticSearch.nuspec

```diff
@@ -11,12 +11,12 @@
     <iconUrl>http://serilog.net/images/serilog-sink-nuget.png</iconUrl>
     <tags>serilog logging elasticsearch</tags>
     <dependencies>
-      <dependency id="Serilog" version="[1.4.204,2)" />
-      <dependency id="Elasticsearch.Net" version="1.1.2" />
+      <dependency id="Serilog" version="[1.5.14,2)" />
+      <dependency id="Elasticsearch.Net" version="1.7.1" />
     </dependencies>
   </metadata>
   <files>
-    <file src="bin40\Release\Serilog.Sinks.Elasticsearch.dll" target="lib\net40" />
-    <file src="bin40\Release\Serilog.Sinks.Elasticsearch.xml" target="lib\net40" />
+    <file src="bin\Release\Serilog.Sinks.Elasticsearch.dll" target="lib\net45" />
```

Comment:
I was not sure how exactly support for net40 is done. Are you saying those two lines should stay as they were? My concern is that the sink's own packages.config (there is only one) always refers to net45 version of Serilog and SerilogFullNetFx, so it seems like what this sink builds as net40 still requires 4.5! 

In short - it seems to me that Serilog.Sinks.Elasticsearch already dropped net40 silently. I may be wrong though.

Do you know what specific nuget command is used to build the package with both binaries? I don't see it among the sources.